### PR TITLE
Next@latest

### PR DESCRIPTION
**Pull request by devharris7**

### CODE REVIEW

**Comment by devharris7** [comment]
seems you can refactor `` with `<Image />` now before merging

**Comment by devharris7** [comment]
image
seems build failed due to the above issues, please fix them as well @Qiming-Liu

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "formik": "^2.2.9",
     "lodash": "^4.17.21",
     "moment": "^2.29.1",
-    "next": "^9.5.3",
+    "next": "^12.1.0",
```
Comment (devharris7) [inline package.json:33]: I forgot to remove `@babel/preset-env` and `@babel/preset-react`

Comment (Qiming-Liu) [reply package.json:33]: ok, i got it

**Comment by devharris7** [comment]
@Qiming-Liu just pushed another commit

**Pull request merged**